fsevent unsupported

(base) PS C:\LarsCode\Peekaboo> npm install  
npm warn old lockfile
npm warn old lockfile The package-lock.json file was created with an old version of npm,
npm warn old lockfile so supplemental metadata must be fetched from the registry.
npm warn old lockfile
npm warn old lockfile This is a one-time fix-up, please be patient...
npm warn old lockfile
npm error code EBADPLATFORM
npm error notsup Unsupported platform for fsevents@1.2.13: wanted {"os":"darwin"} (current: {"os":"win32"})
npm error notsup Valid os:  darwin
npm error notsup Actual os: win32
npm error A complete log of this run can be found in: C:\Users\laliden\AppData\Local\npm-cache\_logs\2024-12-29T18_57_57_749Z-debug-0.log
(base) PS C:\LarsCode\Peekaboo> npm install         
npm error code ERESOLVE
npm error ERESOLVE unable to resolve dependency tree
npm error
npm error While resolving: peekaboo@0.1.0
npm error Found: typescript@3.9.10
npm error node_modules/typescript
npm error   dev typescript@"^3.9.9" from the root project
npm error
npm error Could not resolve dependency:
npm error peer typescript@"2.x" from react-scripts-ts@2.17.0
npm error node_modules/react-scripts-ts
npm error   react-scripts-ts@"2.17.0" from the root project
npm error
npm error Fix the upstream dependency conflict, or retry
npm error this command with --force or --legacy-peer-deps
npm error to accept an incorrect (and potentially broken) dependency resolution.
npm error
npm error
npm error For a full report see:
npm error C:\Users\laliden\AppData\Local\npm-cache\_logs\2024-12-29T18_59_01_156Z-eresolve-report.txt
npm error A complete log of this run can be found in: C:\Users\laliden\AppData\Local\npm-cache\_logs\2024-12-29T18_59_01_156Z-debug-0.log

--- a/package.json
+++ b/package.json
@@ -3,10 +3,10 @@
   "version": "0.1.0",
   "private": true,
   "scripts": {
-    "start": "react-scripts-ts start",
-    "build": "react-scripts-ts build",
-    "test": "react-scripts-ts test --env=jsdom",
-    "eject": "react-scripts-ts eject"
+    "start": "react-scripts start",
+    "build": "react-scripts build",
+    "test": "react-scripts test --env=jsdom",
+    "eject": "react-scripts eject"
   },
   "dependencies": {
     "axios": "^0.28.0",
@@ -18,7 +18,7 @@
     "react-file-picker": "0.0.5",
     "react-google-login": "^5.2.2",
     "react-image-crop": "^6.0.18",
-    "react-scripts-ts": "2.17.0"
+    "react-scripts": "4.0.3"
   },
   "devDependencies": {
     "@types/blob-to-buffer": "^1.2.0",
@@ -31,6 +31,6 @@
     "tslint-config-prettier": "^1.18.0",
     "tslint-config-standard": "^7.0.0",
     "tslint-microsoft-contrib": "^5.0.3",
-    "typescript": "^3.9.9"
+    "typescript": "^4.1.3"
   }
 }

--- a/tsconfig.json
+++ b/tsconfig.json
@@ -8,7 +8,7 @@
     "sourceMap": true,
     "allowJs": true,
     "downlevelIteration": true,
-    "jsx": "react",
+    "jsx": "react-jsx",
     "moduleResolution": "node",
     "rootDir": "src",
     "experimentalDecorators": true,

